src/ui/InputHHMM.tsx

```diff
@@ -0,0 +1,41 @@
+import { Field } from 'formik';
+import styled from '@emotion/styled';
+import { FC } from 'react';
+
+const StyledField = styled(Field)`
+  background: var(--inputbackground);
+  border-bottom: 1px var(--text) solid;
+  color: var(--text);
+  height: 51px;
+  padding-left: 10px;
+  margin-bottom: 10px;
+  font-family: monospace;
+`;
+
+const InputHHMM: FC = (name) => {
```

Comment:
don't be afraid of wordy variable names. and try to avoid acronyms that aren't super widely used.